### PR TITLE
Readmeeeeee

### PR DESCRIPTION
**Pull request by JohnONolan**
IMPROVE. THE THINGS.

Preview: https://github.com/JohnONolan/Ghost/tree/readmeee

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,107 @@
...
+```bash
+npm install -g grunt-cli
+```
 
-If you just want to get a Ghost blog running in the fastest time possible, this method is for you.
+Install Ghost. If you're running locally, use [master](https://github.com/TryGhost/Ghost/tree/master). For production, use [stable](https://github.com/TryGhost/Ghost/tree/stable). :no_entry_sign::rocket::microscope:
 
-For detailed instructions on various platforms, visit the [Ghost Installation Guide](http://support.ghost.org/installation/). If you get stuck, help is available on [our support site](http://support.ghost.org/).
+```bash
+npm install
```
Comment (novaugust) [inline README.md]: Do you want to add
```bash
git clone git://github.com/tryghost/ghost.git
cd ghost
```
before your npm install?

Comment (JohnONolan) [inline README.md]: That would definitely be clear. My question is: would it be clutter? Is it safe to plan that someone following the developer install "(from git)" should know how to git?

Comment (novaugust) [inline README.md]: You should at least say "clone the repo and then..." kind of like how you
said "install node and then...".  You just went from "install grunt
globally! now install all the npm stuff!" <- missing a step if people
turned off their brains and are just reading along (which is what
instructions are for, after all)

Comment (JohnONolan) [inline README.md]: Valid. Updating

**Comment by JohnONolan** [comment]
Dat update.

**Comment by letsjustfixit** [comment]
@JohnONolan may I ask if there is any huge difference between Node 0.10.36 and 0.10.37 as 0.10.36 is preferred? :)

**Pull request merged**